pkg/test/framework/components/environment/kube/cluster.go

```diff
@@ -30,6 +30,7 @@ type Cluster struct {
 	filename            string
 	index               resource.ClusterIndex
 	controlPlaneCluster bool
+	sharesControlPlane  bool
```

Comment:
I don't think we need sharesControlPlane, let's just configure all control plane clusters to expose istiod through ingress.